### ISSUE TITLE
How to build tutorial?

### ISSUE THREAD

**Issue by AlbertoGP** [austral/austral.github.io#61]
The Makefile builds the specification, but not the tutorial.

I’ve got it build something by duplicating `SPEC_SRC` to `TUTORIAL_SRC`, then:
```sh
TUTORIAL_HTML := tutorial/tutorial.html

$(TUTORIAL_HTML): $(TUTORIAL_SRC) _spec/template.html Makefile
	pandoc -t html5 \
		   -f markdown \
		   --template=_spec/template.html \
		   $(PANDOC_FLAGS) --listings --highlight-style=pygments \
		   --mathjax \
		   $(TUTORIAL_SRC) \
		   -o $(TUTORIAL_HTML)
```

This however does not include the example listings read using `{% capture source %}{% include examples/hello-world/hello.aum %}{% endcapture %}` and then `{{ source }}` as is done in `hello-world.md` for instance.

Maybe this version of `pandoc` does not have the necessary modules or whatever, but is there some option I’m missing?

**Comment by AlbertoGP** [comment]
Maybe that’s not `pandoc` but `jekyll` because the `include` directives look like the `liquid` template language they use: https://jekyllrb.com/docs/liquid/

**Comment by eudoxia0** [comment]
Yes, the tutorial is "built" entirely with Jekyll. Running it locally should be enough.
